x-pack/plugins/apm/server/routes/service_groups/get_apm_alerts_client.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { APMRouteHandlerResources } from '../typings';
+
+export type ApmAlertsClient = Awaited<ReturnType<typeof getApmAlertsClient>>;
+
+export async function getApmAlertsClient({
+  plugins,
+  request,
+}: APMRouteHandlerResources) {
+  const ruleRegistryPluginStart = await plugins.ruleRegistry.start();
+  const alertsClient = await ruleRegistryPluginStart.getRacClientWithRequest(
+    request
+  );
+  const apmAlertsIndices = (await alertsClient.getAuthorizedAlertsIndices([
+    'apm',
+  ])) ?? ['.alerts*'];
```

Comment:
When would `alertsClient.getAuthorizedAlertsIndices` return null?

Reply:
nit
```suggestion
  const apmAlertsIndices = (await alertsClient.getAuthorizedAlertsIndices([
    'apm',
  ]));
  
  const index = apmAlertsIndices?.[0] ?? '.alerts*';
```

Reply:
>When would alertsClient.getAuthorizedAlertsIndices return null?

`undefined` is on of the return types, and it looks like typescript resolves it as undefined when it `throw`s